### DISCUSSION TITLE
What is the command to spawn in an enchantment with a book?

### DISCUSSION THREAD

**Discussion by Maxmarsride** [Geolykt/EnchantmentsPlus#140] [Q&A] [answered]
i want to know the command to make it into a book forum so i can sell enchantments in a shop? if there is any way please do let me know :)

**Comment by Geolykt** [comment]
The command `/ench give <player> <material> <enchantment> <level>` would probably be the most appropriate for this usecase, but it only supports a single enchantment. Alternatively as long as you use a lore-based enchantment getter (the default, albeit not recommended) you can use the vanilla give command and add the relevant lore entires (don't ask me how that works, I just know that it does). For the PDC-based enchantment getters (not the default, but recommended) you'd need to alter the NBT data which is a bit more complex but still ultimately doable

**Reply by Maxmarsride** [reply]
so if i were to let's say set it up so i can sell eternal enchantment the one that is unbreaking for 25 bucks i got to set it up so it writes in the console "give %USERNAME& minecraft:enchantedbook" but how would it be for console based so it fully automates it.?

**Reply by Maxmarsride** [reply, marked as answer]
nevermind... it's /ench give Maxmarsride BOOK Siphon 4 okay got it. accept replace my name with %username% for the tebex command

**Reply by Geolykt** [reply]
You can also use selectors instead of just the username - so `@a` would be valid (meaning every online player).
The player UUID is also valid as a logical consequence iirc

**Reply by Maxmarsride** [reply]
yeah but for automating it i need to give it %username% as a permission node.
